algebra/order_functions.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2017 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
-import algebra.ordered_group order.lattice
+import algebra.ordered_group order.lattice tactic.wlog
```

Comment:
Can we avoid `wlog` here? I hate to ask for this but `wlog` has `data.list.perm` as a requirement, so this is going to make the dependency structure all tangled up.

Reply:
I thought you might ask for that. Yeah, it can be avoided without too much work.